common/tier4_planning_rviz_plugin/include/tier4_planning_rviz_plugin/path/display_base.hpp

```diff
@@ -47,39 +49,6 @@
 
 namespace
 {
-std::unique_ptr<Ogre::ColourValue> gradation(
-  const QColor & color_min, const QColor & color_max, const double ratio)
-{
-  std::unique_ptr<Ogre::ColourValue> color_ptr(new Ogre::ColourValue);
-  color_ptr->g =
-    static_cast<float>(color_max.greenF() * ratio + color_min.greenF() * (1.0 - ratio));
-  color_ptr->r = static_cast<float>(color_max.redF() * ratio + color_min.redF() * (1.0 - ratio));
-  color_ptr->b = static_cast<float>(color_max.blueF() * ratio + color_min.blueF() * (1.0 - ratio));
-
-  return color_ptr;
-}
-
-std::unique_ptr<Ogre::ColourValue> setColorDependsOnVelocity(
-  const double vel_max, const double cmd_vel)
-{
-  const double cmd_vel_abs = std::fabs(cmd_vel);
-  const double vel_min = 0.0;
-
-  std::unique_ptr<Ogre::ColourValue> color_ptr(new Ogre::ColourValue());
-  if (vel_min < cmd_vel_abs && cmd_vel_abs <= (vel_max / 2.0)) {
-    double ratio = (cmd_vel_abs - vel_min) / (vel_max / 2.0 - vel_min);
-    color_ptr = gradation(Qt::red, Qt::yellow, ratio);
-  } else if ((vel_max / 2.0) < cmd_vel_abs && cmd_vel_abs <= vel_max) {
-    double ratio = (cmd_vel_abs - vel_max / 2.0) / (vel_max - vel_max / 2.0);
-    color_ptr = gradation(Qt::yellow, Qt::green, ratio);
-  } else if (vel_max < cmd_vel_abs) {
-    *color_ptr = Ogre::ColourValue::Green;
-  } else {
-    *color_ptr = Ogre::ColourValue::Red;
-  }
-
-  return color_ptr;
-}
```

Comment:
Why did you move these to protected?
If it's not important, could you move them back to reduce the diff?

Reply:
build was complaining about them not having access to properties, I moved them to protected so they can access the class properties like property_min_color_ and still be available for any subclasses that might need them.

Reply:
Applied with: https://github.com/autowarefoundation/autoware.universe/pull/8972/commits/d8ba16f9a28d27cc156a64f8cd64e15b0e96d58c